src/Controllers/CompaniesController.php

```diff
@@ -49,7 +49,7 @@ public function searchAction(): void
                 $this->request->getPost()
             );
 
-            $this->persistent->searchParams = $query->getParams();
+            $this->persistent->searchParams = array('di'=>null) + $query->getParams();
```

Comment:
Can you use the newer syntax (short) for the array please and also run it through codesniffer (spaces needed before and after `=>`)

Reply:
I have corrected it as requested.